docs/user-interface/handlers/customize.md

```diff
@@ -61,253 +52,202 @@ public partial class CustomizeEntryPage : ContentPage
-| `CheckBox` | `ICheckBox` | `CheckBoxHandler` | `Mapper` |
-| `CollectionView` |  | `CollectionViewHandler` | `Mapper` |
-| `ContentView` | `IContentView` | `ContentViewHandler` | `Mapper` |
-| `DatePicker` | `IDatePicker` | `DatePickerHandler` | `Mapper` |
-| `Editor` | `IEditor` | `EditorHandler` | `Mapper` |
-| `Ellipse` | | `ShapeViewHandler` | `Mapper` |
-| `Entry` | `IEntry` | `EntryHandler` | `Mapper` |
-| `GraphicsView` | `IGraphicsView` | `GraphicsViewHandler` | `Mapper` |
-| `Image` | `IImage` | `ImageHandler` | `Mapper` |
-| `ImageButton` | `IImageButton` | `ImageButtonHandler` | `Mapper` |
-| `IndicatorView` | `IIndicatorView` | `IndicatorViewHandler` | `Mapper` |
-| `Label` | `ILabel` | `LabelHandler` | `Mapper` |
-| `Line` | | `LineHandler` | `Mapper` |
-| `Path` | | `PathHandler` | `Mapper` |
-| `Picker` | `IPicker` | `PickerHandler` | `Mapper` |
-| `Polygon` | | `PolygonHandler` | `Mapper` |
-| `Polyline` | | `PolylineHandler` | `Mapper` |
-| `ProgressBar` | `IProgress` | `ProgressBarHandler` | `Mapper` |
-| `RadioButton` | `IRadioButton` | `RadioButtonHandler` | `Mapper` |
-| `Rectangle` | | `RectangleHandler` | `Mapper` |
-| `RefreshView` | `IRefreshView` | `RefreshViewHandler` | `Mapper` |
-| `RoundRectangle` | | `RoundRectangleHandler` | `Mapper` |
-| `ScrollView` | `IScrollView` | `ScrollViewHandler` | `Mapper` |
-| `SearchBar` | `ISearchBar` | `SearchBarHandler` | `Mapper` |
-| `Slider` | `ISlider` | `SliderHandler` | `Mapper` |
-| `Stepper` | `IStepper` | `StepperHandler` | `Mapper` |
-| `SwipeView` | `ISwipeView` | `SwipeViewHandler` | `Mapper` |
-| `Switch` | `ISwitch` | `SwitchHandler` | `Mapper` |
-| `TimePicker` | `ITimePicker` | `TimePickerHandler` | `Mapper` |
-| `WebView` | `IWebView` | `WebViewHandler` | `Mapper` |
-
-All handlers are in the `Microsoft.Maui.Handlers` namespace, with the following exceptions:
-
-- `CarouselViewHandler` and `CollectionViewHandler` are in the `Microsoft.Maui.Controls.Handlers.Items` namespace.
-- `LineHandler`, `PathHandler`, `PolygonHandler`, `PolylineHandler`, `RectangleHandler`, and `RoundRectangleHandler` are in the `Microsoft.Maui.Controls.Handlers` namespace.
-
-The interfaces in the table above are in the `Microsoft.Maui` namespace.
-
-<!-- Remove the text above once their are API docs that can be linked into -->
-
-## Renderer-based views
-
-The following legacy Xamarin.Forms views are backed by renderers, rather than handlers, and use a different customization approach:
-
-- `BoxView`
-- `Frame`
-- `ListView`
-- `TableView`
+The advantage of this approach is that conditional compilation isn't required, and that the partial methods don't have to be implemented on each platform. If an implementation isn't provided on a platform, then the method and all calls to the method are removed at compile time. For information about partial methods, see [Partial methods](/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods#partial-methods).
```

Comment:
I feel like there should also be a link to the article that talks about platform specific code, folder layout, naming rules, and project settings.